components/Drawer/Drawer.tsx

```diff
@@ -0,0 +1,36 @@
+import classNames from 'classnames';
+
+export type DrawerPropsType = {
+  /**
+   * Content to be rendered in the Drawer.
+   */
+  children: React.ReactNode;
+  /**
+   * Applies class names to the container element.
+   */
+  className?: string;
+  /**
+   * Sets if the content is visible.
+   * @default false
+   */
+  isVisible?: boolean;
+};
+
+function Drawer({ children, className, isVisible = false }: DrawerPropsType) {
+  return (
+    <div
+      className={classNames(
+        className,
+        'hidden lg:block transition-all ease-in-out duration-1000 fixed top-0 bottom-0  overflow-hidden width-full z-[2]',
```

Comment:
```suggestion
        'hidden lg:block transition-all ease-in-out duration-1000 fixed top-0 bottom-0 overflow-hidden width-full z-[2]',
```
same

Reply:
There doesn't appear to be a change in the suggestion. Am I missing a tiny detail?

Reply:
there are 2 spaces between `bottom-0` and `overflow-hidden`. Perhaps you need to lower the contrast on your monitor?

<img width="757" alt="Screenshot 2024-03-05 at 3 05 09 PM" src="https://github.com/OperationCode/front-end/assets/9523719/973f6ba6-2cdf-49b2-85cc-c7e4245f8294">

Reply:
Ahhhh...  I actually did see it, but didn't know what it was. This is merged, should I make that change in one of the other branches I am working?